Anlab.Mvc/Views/TestItems/Delete.cshtml

```diff
@@ -0,0 +1,63 @@
+@model Anlab.Core.Domain.TestItem
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Delete</title>
+</head>
+<body>
+    <h2>Delete Test Item</h2>
+
+<div>
+    <hr />
+    <dl class="dl-horizontal">
+        <h3>Are you sure you want to delete this?</h3>
+
+        <dt>
+            @Html.DisplayNameFor(model => model.Id)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Id)
+        </dd>        <dt>
+            @Html.DisplayNameFor(model => model.Analysis)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Analysis)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Category)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Category)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Group)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Group)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Public)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Public)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Notes)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Notes)
+        </dd>
+    </dl>
```

Comment:
@laholstege Consider making a partial view for these common display and edit fields.